fbmessenger/__init__.py

```diff
@@ -288,17 +289,20 @@ def handle(self, payload):
                     return self.read(message)
 
     def get_user(self, fields=None, timeout=None):
-        return self.client.get_user_data(self.last_message, fields=fields, timeout=timeout)
+        return self.client.get_user_data(self.get_user_id(), fields=fields, timeout=timeout)
 
     def send(self, payload, messaging_type='RESPONSE', notification_type='REGULAR', timeout=None, tag=None):
-        return self.client.send(payload, self.last_message, messaging_type=messaging_type,
+        return self.client.send(payload, self.get_user_id(), messaging_type=messaging_type,
                                 notification_type=notification_type, timeout=timeout, tag=tag)
 
     def send_action(self, sender_action, timeout=None):
-        return self.client.send_action(sender_action, self.last_message, timeout=timeout)
+        return self.client.send_action(sender_action, self.get_user_id(), timeout=timeout)
 
     def get_user_id(self):
```

Comment:
@rickydunlop what do you think about that?

Reply:
I'm thinking this should raise an error here, or at least log something, if we can't get the ID. 
The ID is required in `send_action` method so sending `None` is just going to cause an error later on. 
What do you think?

Reply:
Ok, we raise an exception instead of returning None in get_user_id.